packages/manager/apps/dedicated/client/app/network-security/scrubbing-center/scrubbing-center.controller.js

```diff
@@ -0,0 +1,114 @@
+import { PERIODS, PERIOD_LIST } from './scrubbing-center.constant';
+
+export default class ScrubbingCenterController {
+  /* @ngInject */
+  constructor($http, $translate, Alerter, scrubbingCenterService) {
+    this.$http = $http;
+    this.$translate = $translate;
+    this.Alerter = Alerter;
+    this.scrubbingCenterService = scrubbingCenterService;
+    this.PERIODS = PERIODS;
+    this.PERIOD_LIST = PERIOD_LIST;
+  }
+
+  $onInit() {
+    this.errorMessage = '';
+    this.pageSize = 10;
+    this.initPeriods();
+    this.initService();
+    this.getAllEvents();
+  }
+
+  initPeriods() {
+    this.periods = this.PERIODS.map((period) => ({
+      name: period.key,
+      label: this.$translate.instant(period.value),
+    }));
+    [this.period] = this.periods;
+  }
+
+  initService() {
+    return this.$http
+      .get('/sws/products/services', {
+        serviceType: 'aapi',
+      })
+      .then(({ data }) => {
+        this.services = data.sort(({ serviceName: a }, { serviceName: b }) =>
+          a > b ? 1 : -1,
+        );
+      });
+  }
+
+  getAllEvents(service) {
+    this.isLoading = true;
+    this.events = [];
+    let after = '';
+    switch (this.period.name) {
+      case this.PERIOD_LIST.last24h:
+        after = moment()
+          .subtract(1, 'days')
+          .toISOString();
+        break;
+      case this.PERIOD_LIST.lastMonth:
+        after = moment()
+          .subtract(1, 'months')
+          .toISOString();
+        break;
+      case this.PERIOD_LIST.lastWeek:
+        after = moment()
+          .subtract(7, 'days')
+          .toISOString();
+        break;
+      case this.PERIOD_LIST.lastYear:
+        after = moment()
+          .subtract(1, 'years')
+          .toISOString();
+        break;
+      default:
+        after = moment()
+          .subtract(1, 'days')
+          .toISOString();
```

Comment:
```suggestion
      case this.PERIOD_LIST.lastMonth:
        after = moment()
          .subtract(1, 'months')
          .toISOString();
        break;
      case this.PERIOD_LIST.lastWeek:
        after = moment()
          .subtract(7, 'days')
          .toISOString();
        break;
      case this.PERIOD_LIST.lastYear:
        after = moment()
          .subtract(1, 'years')
          .toISOString();
        break;
      default:
        after = moment()
          .subtract(1, 'days')
          .toISOString();
```
No need case 24hours, because it's same that default case.